docusaurus/docs/developing-components-in-isolation.md

```diff
@@ -41,6 +41,8 @@ Learn more about React Storybook:
 
 Styleguidist combines a style guide, where all your components are presented on a single page with their props documentation and usage examples, with an environment for developing components in isolation, similar to Storybook. In Styleguidist you write examples in Markdown, where each code snippet is rendered as a live editable playground.
 
+![React Styleguidist Demo](https://user-images.githubusercontent.com/1703219/74945569-51c6ad00-543b-11ea-8351-f4d86860893a.gif)
```

Comment:
Nice catch, shall we also move the demo for Storybook to the **Getting Started with Storybook** section? 
https://github.com/facebook/create-react-app/pull/9761/files#diff-fa43576dbc34a19ae29831d54c0253b1R17

In this way, it's a bit more coherent.

Reply:
Good call, added 1f8d8c7. Used the gh ui so let me know if i need to squash.